opencti-platform/opencti-front/src/private/components/settings/sub_types/Root.tsx

```diff
@@ -21,27 +16,15 @@ export const subTypeQuery = graphql`
 
 const RootSubType = () => {
   const { subTypeId } = useParams() as { subTypeId: string };
+
+  const data = useLazyLoadQuery<RootSubTypeQuery>(subTypeQuery, { id: subTypeId });
```

Comment:
using QueryRenderer caused a full page refresh every time one setting is changed  (not only the settings related to layout)